new/index.md

```diff
@@ -7,6 +7,19 @@ What's been New on the [arXiv.org](/) e-print archives
 
 (Announcements since Summer 1996. See also more haphazard collections of messages from [1994-1996](/new/94-96) and [1991-1994](/new/91-94).)
 
+#### June 2019
+
+<span id="june11_2019"></span>
+11 June 2019
+
+**arXiv created a Systems and Control category eess.SY that will merge with cs.SY. We have also merged the existing Numerical Analysis categories math.NA/cs.NA**
+
+Two changes to arXiv categories were made today. Thank you to arXiv moderators, subject advisory committees, and scientific advisory board members who provided leadership and input on these changes. 
```

Comment:
I suggest putting the the thanks/acknowledgement at the end in its own paragraph, so that the important info it up front and clear. So I suggest moving, "Thank you to arXiv moderators, subject advisory committees, and scientific advisory board members who provided leadership and input on these changes," at the end, and moving the sentence, "Special thanks to our math.NA and cs.NA moderators for putting in extra effort over the past two month to pilot this move," too.

Reply:
Suggested changes made.